### PR TITLE
Update quick-start-with-tidb.md

### LINKED ISSUE #7328: missing global variables declaration
File: [/quick-start-with-tidb.md](https://docs.pingcap.com/zh/tidb/stable/quick-start-with-tidb)

下载并安装 TiUP：
curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh

安装 TiUP 的 cluster 组件：
tiup cluster

tiup安装单机步骤， 1，2中间还差一个：source .bash_profile

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
add a step to source bash_profile when we set up a cluster on linux

### Which TiDB version(s) do your changes apply to? (Required)
after 4.0

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?
fix #7328 

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lilyjazz** [comment]
#7328

**Comment by lilyjazz** [comment]
@AstroProfundis @srstack PTAL

**Review by AstroProfundis — APPROVED** [review]

**Comment by TomShawn** [comment]
@ran-huang Please be aware of this change in quick start.

**Review by srstack — COMMENTED** [review]
/LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -272,23 +272,35 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
     curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
     ```
 
-2. 安装 TiUP 的 cluster 组件：
+2. 声明全局环境变量。
```
Comment (TomShawn) [inline quick-start-with-tidb.md:275]:
```suggestion
2. 声明全局环境变量：
```

**Pull request merged**